README.md

```diff
@@ -1 +1,24 @@
-# llm_validation
\ No newline at end of file
+## HOW TO RUN:
+### 1. Create constants.py file with:
+- PROMPT
+- OPTIONS
+- INSTRUCTION
+- INPUT_TEXT_1
+- INPUT_TEXT_2
+- ...
+- INPUT_TEXT_9
+
+### 2. Create .env file with:
+- OPENROUTER_API_KEY
+- DEFAULT_LLM_MODEL
+- DEFAULT_LLM_TEMPERATURE
+- CSV_FILE_PATH
```

Comment:
what is CSV file? file names should be descriptiver and examples should be in the repo. Right now I can't understand what is expected to be in this file, so when we have other developers looking at this code they will not understand it for sure.

Reply:
I got. Now will create env.sample and csv_file sample

Reply:
it's not only about samples, it's about description of the purpose of the file. This file should be named DATA_SET_CSV_FILE_PATH, if it's a path to data set file.